uint8 i_sensor_id
uint16 i_min_fps
uint16 camera_fps_override
uint16 i_src_width
uint16 i_src_height
uint8 i_enc_variant
uint32 i_split_duration
uint16 i_log_level
string ac_log_file

uint8 Record_e_enc_type
uint32 Record_write_bitrate
uint16 Record_write_width_override
uint16 Record_write_height_override
uint8 Record_i_preset_level
uint8 Record_i_profile
uint32 Record_i_i_interval
uint8 Record_i_num_b
int64 Record_i_min_qp_i
int64 Record_i_max_qp_i
int64 Record_i_min_qp_p
int64 Record_i_max_qp_p
int64 Record_i_min_qp_b
int64 Record_i_max_qp_b
uint8 Record_i_rate_control
string Record_ac_output_file

uint8 Stream_e_enc_type
uint32 Stream_write_bitrate
uint16 Stream_write_width_override
uint16 Stream_write_height_override
uint8 Stream_i_preset_level
uint8 Stream_i_profile
uint32 Stream_i_i_interval
uint8 Stream_i_num_b
int64 Stream_i_min_qp_i
int64 Stream_i_max_qp_i
int64 Stream_i_min_qp_p
int64 Stream_i_max_qp_p
int64 Stream_i_min_qp_b
int64 Stream_i_max_qp_b
uint8 Stream_i_rate_control
uint32 Stream_i_port
string Stream_ac_output_ip

uint8 Detect_e_enc_type
uint32 Detect_write_bitrate
uint16 Detect_write_width_override
uint16 Detect_write_height_override
uint8 Detect_i_preset_level
uint8 Detect_i_profile
uint32 Detect_i_i_interval
uint8 Detect_i_num_b
int64 Detect_i_min_qp_i
int64 Detect_i_max_qp_i
int64 Detect_i_min_qp_p
int64 Detect_i_max_qp_p
int64 Detect_i_min_qp_b
int64 Detect_i_max_qp_b
uint8 Detect_i_rate_control
uint32 Detect_i_port
string Detect_ac_output_ip
string Detect_ac_output_file
